README.md

```diff
@@ -48,7 +52,12 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+
+town[:residents]delete.["Belle"]
+town[:castle][:guests]push.()"Belle")
```

Comment:
syntax error - you're very close! try out these methods in pry before submitting in the future